tests/functional/test_libraries.py

```diff
@@ -87,6 +87,7 @@ def test_gevent(pyi_builder):
         """)
 
 
+@xfail(is_py36, reason='Fails on python 3.6')
```

Comment:
These fail for the same reason pytest-xdist/execnet was failing, the depreciation of `inspect.getargspec()` in python 3.6. See https://travis-ci.org/pyinstaller/pyinstaller/jobs/119229568#L1527.